app/models/damage.rb

```diff
@@ -0,0 +1,24 @@
+class Damage < ActiveRecord::Base
```

Comment:
But why is a damage reservation even needed? Isn't the presence of a `Damage` object on an item enough to signify that an item can't be reserved? Why would a reservation need to be involved?